src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2010,56 +1990,31 @@ because xbuild doesn't support framework reference assemblies.
 	</ItemGroup>
 </Target>
 
-<Target Name="_CopyPdbFiles"
-		Inputs="@(_ResolvedPortablePdbFiles)"
-		Outputs="$(_AndroidStampDirectory)_CopyPdbFiles.stamp"
-		DependsOnTargets="_ConvertPdbFiles">
-	<ItemGroup>
-		<_CopyPdbFilesLinkerFiles Include="@(_ResolvedPortablePdbFiles->'$(MonoAndroidLinkerInputDir)%(Filename)%(Extension)')" />
-	</ItemGroup>
-	<CopyIfChanged SourceFiles="@(_ResolvedPortablePdbFiles)" DestinationFiles="@(_CopyPdbFilesLinkerFiles)" />
-	<Touch Files="$(_AndroidStampDirectory)_CopyPdbFiles.stamp" AlwaysCreate="True" />
-	<ItemGroup>
-		<FileWrites Include="@(_CopyPdbFilesLinkerFiles)" />
-	</ItemGroup>
-</Target>
-
-<Target Name="_CopyMdbFiles"
-		Inputs="@(_ResolvedMdbFiles)"
-		Outputs="$(_AndroidStampDirectory)_CopyMdbFiles.stamp"
-		DependsOnTargets="_CollectMdbFiles">
-	<ItemGroup>
-		<_CopyMdbFilesLinkerFiles Include="@(_ResolvedMdbFiles->'$(MonoAndroidLinkerInputDir)%(Filename)%(Extension)')" />
-	</ItemGroup>
-	<CopyIfChanged SourceFiles="@(_ResolvedMdbFiles)" DestinationFiles="@(_CopyMdbFilesLinkerFiles)" />
-	<Touch Files="$(_AndroidStampDirectory)_CopyMdbFiles.stamp" AlwaysCreate="True" />
-	<ItemGroup>
-		<FileWrites Include="@(_CopyMdbFilesLinkerFiles)" />
-	</ItemGroup>
-</Target>
-	
 <Target Name="_LinkAssemblies"
   DependsOnTargets="_ResolveAssemblies;_CreatePackageWorkspace;$(_BeforeLinkAssemblies);_GenerateJniMarshalMethods;_LinkAssembliesNoShrink;_LinkAssembliesShrink" />
 
 <Target Name="_GenerateJniMarshalMethods"
     Condition="'$(AndroidGenerateJniMarshalMethods)' == 'True' And '$(AndroidLinkMode)' != 'None' And '$(OS)' != 'Windows_NT'"
     DependsOnTargets="_GetReferenceAssemblyPaths;_SetLatestTargetFrameworkVersion"
-    Inputs="@(ResolvedUserAssemblies->'$(MonoAndroidLinkerInputDir)%(Filename)%(Extension)')"
-    Outputs="$(_AndroidJniMarshalMethodsFlag)">
-  <ItemGroup>
-    <_JniFrameworkAssembly Include="Mono.Android.dll" />
-    <_JniFrameworkAssembly Include="OpenTK-1.0.dll" />
-    <_JniFrameworkAssembly Include="OpenTK.dll" />
-    <_JniFrameworkAssembly Include="Xamarin.Android.NUnitLite.dll" />
-  </ItemGroup>
```

Comment:
We should still filter SDK assemblies and only process the above listed ones.

Reply:
I found this `Condition` was basically always `True`, so I just removed it: https://github.com/xamarin/xamarin-android/blob/9efb5f97e3afaa39a38d113a15e471e038b850e5/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L2057

If we don't need to pass this assemblies though, I'll fix that. 👍

Reply:
It is the other way around. The condition is only true for assemblies in `_JniFrameworkAssembly`. It filters out the rest of SDK assemblies as we don't need to process them.

Reply:
Because the syntax confused me and @dellis1972, I flipped it around a little, but the result is the same, thanks:
```xml
<_AssembliesToProcess Include="@(ResolvedFrameworkAssemblies)" Condition=" '%(Filename)' == '@(_JniFrameworkAssembly->'%(Filename)')' " />
```

Reply:
The nested `'` look a bit confusing, it should work as well though. It should still "batch" the inclusion on outer join of the groups.